### PR TITLE
Improvements on cepher

### PR DESCRIPTION
**Pull request by jairsjunior**
List of Improvements:

-> Add of go modules dependency manager
-> Added tests cases
-> Change manager of shell execution
-> Correction of create volume at swarm deploy, including lock at the Get information

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,11 +47,4 @@ COPY --from=BUILD /go/bin/* /bin/
 ADD startup.sh /
 ADD ceph.conf.template /
 
-# VOLUME [ "/run/docker/plugins" ]
```
Comment (flaviostutz) [inline Dockerfile]: Please, leave the commented volumes as they are used for documentation purposes of which mount points have to be set

File: Dockerfile [hunk 2]
```diff
@@ -43,11 +47,4 @@ COPY --from=BUILD /go/bin/* /bin/
...
-# VOLUME [ "/dev" ]
```
Comment (flaviostutz) [inline Dockerfile]: Same here

File: Dockerfile (hunk 1 above)
Comment (flaviostutz) [reply Dockerfile]: @tiagostutz, please check what do you think about this.

**Comment by jairsjunior** [comment]
@flaviostutz, i made the suggested revisions, please if could you verify if everything is okay to accept the pull request.

**Pull request merged**